Automation/Mobile/qama.md

```diff
@@ -108,4 +107,47 @@ The idea of a test is to verify some result. Therefore, each test must contain a
 For more info on assertions, read:
 
 - [Writing assertions](https://beta.infinum.com/handbook/qa/automation/web/selenium-and-qawa#writing-assertions)
-- [Asserts](https://infinum.com/handbook/qa/automation/general/way-of-working#asserts)
\ No newline at end of file
+- [Asserts](https://infinum.com/handbook/qa/automation/general/way-of-working#asserts)
+
+
+## Appium server logs
+
+Server logs can be helpful when debugging Appium issues. Due to potential security issues, the option is disabled by default. 
+If you want to have server logs available during test runs, start Appium server by running: 
+
+```
+appium --allow-insecure=get_server_logs
+```
+
+**NOTE**:
+
+- Use the `--allow-insecure` option only if you are running the server in a secure environment.
+
+Read [Appium Security](https://appium.io/docs/en/writing-running-appium/security/index.html#insecure-features) for more information.
+
+
+### Enable saving server logs in QAMA
+
+The method for saving the logs is already implemented in `QAMA`.
+
+There are a few more steps to take to finish the implementation. 
+
+- Add `logs` folder to the project root
+    - This folder will store the generated logs
+
+
+- Call the `__save_logs` method in the `driver` fixture in the `conftest.py`
```

Comment:
I guess that you can omit "the" from `the conftest.py` (or add "file" at the end: `the conftest.py file`)